Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -126,6 +126,16 @@
   category: Engineering
   group: market
 
+- type: cargoProduct
```

Comment:
Same here with our own custom product entries, lets keep them separate in a new file in the _nf folder